server/mongodb/actions/Tag.js

```diff
@@ -0,0 +1,89 @@
+import mongoDB from "../index";
+import Tag from "../models/Tag";
+
+export async function createTag({ name }) {
+  await mongoDB();
+  name = name.toLowerCase();
+  const tag = await Tag.create({ name });
+  return tag;
+}
+
+export async function getTags() {
+  await mongoDB();
+  const tag = await Tag.find({}).sort({ name: 1 });
+  return tag;
+}
+
+export async function getTagsObject() {
+  await mongoDB();
+  // const tag = Tag.find().sort({ name: 1 });
+  const tag = await Tag.aggregate([
```

Comment:
Bro can you explain this code to me sometime? I got an interview coming up and they'll probably ask MongoDB questions...